### PR TITLE
BZ1903373: Explain the "-2" flag in the Configuring Linuxptp services Section

### PR DESCRIPTION
**Pull request by xenolinux**
Fixes: [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1903373)
Preview
QA contact: @zhaozhanqi @huiran0826 
OCP version: 4.6, 4.7, 4.8, 4.9, 4.10

### CODE REVIEW

**Review by jeana-redhat — COMMENTED** [review]
LGTM!

**Comment by xenolinux** [comment]
@jboxman should I wait until PR[1] merges?
[1] https://github.com/openshift/openshift-docs/pull/39536

File: modules/nw-ptp-configuring-linuxptp-services-as-ordinary-clock.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@ spec:
 <2> Specify an array of one or more `profile` objects.
 <3> Specify the name of a profile object which uniquely identifies a profile object.
 <4> Specify the network interface name to use by the `ptp4l` service, for example `ens787f1`.
-<5> Specify system config options for the `ptp4l` service, for example `-s -2`. The options should not include the network interface name `-i <interface>` and service config file `-f /etc/ptp4l.conf` because the network interface name and the service config file are automatically appended.
+<5> Specify system config options for the `ptp4l` service and select the IEEE 802.3 network transport, for example `-2`. The options should not include the network interface name `-i <interface>` and service config file `-f /etc/ptp4l.conf` because the network interface name and the service config file are automatically appended.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/nw-ptp-configuring-linuxptp-services-as-ordinary-clock.adoc]: Addressed

**Review by SchSeba — COMMENTED** [review]
/lgtm

**Pull request merged**